news/_posts/releases/2022-01-26-prestashop-1-7-8-3-maintenance-release.md

```diff
@@ -0,0 +1,84 @@
+- --
+layout: post
+title:  "Release Of PrestaShop 1.7.8.3"
+subtitle: "MAINTENANCE VERSION OF THE 1.7.8.X BRANCH"
+date:   2022-01-24 14:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2021/12/1.7.8.3_banner.jpg
+icon: icon-checked
+tags:
+- version
+- patch
+- releases
+
+---
+
+PrestaShop 1.7.8.3 is now available. This maintenance release fixes 25 bugs and 1 secrurity issue since the 1.7.8.2 release.
+Note : 2 of this fixes break the backward compatibility ( ([#26933][https://github.com/PrestaShop/PrestaShop/issues/26933](https://github.com/PrestaShop/PrestaShop/issues/26933)) and ([#26643][https://github.com/PrestaShop/PrestaShop/issues/26643](https://github.com/PrestaShop/PrestaShop/issues/26643)) ) 
```

Comment:
```suggestion
Note : two of these fixes are not fully backward compatible ( [#26933](https://github.com/PrestaShop/PrestaShop/issues/26933) and [#26643](https://github.com/PrestaShop/PrestaShop/issues/26643) ) 
```

Bug https://github.com/PrestaShop/PrestaShop/issues/26933 was fixed by https://github.com/PrestaShop/autoupgrade/pull/439 on autoupgrade module so is it really an issue?